contrib/raftexample/raft.go

```diff
@@ -103,8 +103,9 @@ func (rc *raftNode) publishEntries(ents []raftpb.Entry) bool {
 		case raftpb.EntryConfChange:
 			var cc raftpb.ConfChange
 			cc.Unmarshal(ents[i].Data)
-
-			rc.node.ApplyConfChange(cc)
+			if rc.node != nil {
```

Comment:
instead of doing this, we probably should replay log after we start/restart the node?